src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -50,7 +50,8 @@ public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone
             @JsonProperty("email") String email, @JsonProperty("address") String address,
             @JsonProperty("gender") String gender, @JsonProperty("birthdate") String birthdate,
             @JsonProperty("race") String race, @JsonProperty("religion") String religion,
-            @JsonProperty("survey") String survey, @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
+            @JsonProperty("survey") List<JsonAdaptedSurvey> survey,
```

Comment:
Wondering if this should be changed to surveys for both the name of the List and the JsonProperty. Will have to refactor all the JSON files though.

Reply:
You're right, `surveys` is better. Shouldn't be too much of a problem factoring JSON files.

Reply:
Renamed `survey` to `surveys` as suggested.